src/core/Board.js

```diff
@@ -38,33 +36,27 @@ export const Board = () => {
           }
           setInputTracker([]);
         }
-      }, 500)
+      }, 500);
     } else {
       console.log("no way buddy!");
     }
   };
 
-  return ( <
-    div className = {
-      styles.board
-    } > {
-      shuffledCards.map((card, i) => ( <
-        Card card = {
-          card
-        }
-        key = {
-          i
-        }
-        onClick = {
-          () => {
+  return (
+    <div className={styles.board}>
+      {" "}
+      {shuffledCards.map((card, i) => (
+        <Card
+          card={card}
+          key={i}
+          onClick={() => {
             handleFlip(i, card);
-          }
-        } > {
-          card.face
-        } <
-        /Card>
-      ))
-    } <
-    /div>
+          }}
+        >
+          {" "}
```

Comment:
Do you need the `{" "}`? Or, is it something your editor added?

Reply:
Ooops! My editor added this.